src/pages/build-apps/guides/data-storage.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Data storage
+description: Save and retrieve data for users with Gaia
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/gears.svg
+  sm: /images/gears.svg
+---
+
+## Introduction
+
+This guide explains how to save and retrieve data for users with [Gaia](/build-apps/references/gaia) by implementing the [`connect`](https://github.com/blockstack/ux/tree/master/packages/connect#stacksconnect) and [`storage`](https://github.com/blockstack/ux/tree/master/packages/storage#stacksstorage) packages of Stacks.js.
+
+Data storage provides a way for users to save both public and private data off-chain while retaining complete control over it.
+
+Storing data off the Stacks blockchain ensures that apps can provide users with high performance and high availability for data reads and writes without the involvement of centralized parties that could compromise their privacy or accessibility.
+
+[See the Todos app tutorial](/build-apps/tutorials/todos) for a concrete example of this functionality in practice.
+
+## Install dependencies
+
+The following dependencies must be installed:
+
+```
+npm install @stacks/connect
```

Comment:
Let's use just one command for both: `npm install @stacks/connect @stacks/storage`